README.md

```diff
@@ -152,5 +152,20 @@ $ echo $?
 0
 ```
 
+## VPN Modules
+
+We have added two modules which can be used to identify VPN servers using OpenVPN over TCP, SSTP and PPTP.
+
+The SSTP module performs a TLS handshake, sends out an SSTP_DUPLEX_POST HTTP request and logs the server response. Other than the targets, no additional flag needs to be specified for the SSTP module to work. The default port is 443.
+
+The TCP module can be used to establish a TCP connection over which a specified probe will be sent to the target. The module includes features for an easier support of OpenVPN and PPTP probes. The default port is 443 and might have to be specified depending on the protocol (e.g. 1723 for PPTP). The following flags can be used in combination with the TCP module:
+
+ * `--hex`: Store response in hex value.
+ * `--keymethod`: Specifies OpenVPN key method. This can be eith 1 or 2.
+ * `--protocol`: Specifies which VPN protocol to scan for. Allows us to omit a probe. This can be either *`openvpn`* or *`pptp`*.
+ * `--hmac`: Specify this flag to include HMAC, packet ID and timestamp in an OpenVPN request.
+ * `--probefile`: File which contains a custom probe to be sent over the TCP connection.
```

Comment:
It makes sense to add a sample probefile to the repo, either here in the readme same as what you have for `multiple.ini` , or in just a separate file and a reference here in readme.

Reply:
Since the probefile does not require a particular format and only contains the probe, I added a small example in the README.md VPN section instead of adding an example file.

Reply:
Looks good.